chore(variables): ensure we cover "empty source mapping to null (#7109)
camunda/camunda-docs: docs/components/concepts/variables.md

Variable mappings are defined in the process as extension elements under `ioMapping`. Every variable mapping has a `source` and a `target` expression.

> The `source` expression defines the **value** of the mapping. Usually, it [accesses a variable](/components/modeler/feel/language-guide/feel-variables.md#access-variable) of the process instance that holds the value. If the variable or the nested property doesn't exist, it uses `null` as the value. Same applies if you do not provide a `source`.

Reviewer: ```suggestion
The `source` expression defines the **value** of the mapping. It usually [accesses a variable](/components/modeler/feel/language-guide/feel-variables.md#access-variable) of the process instance that holds the value. If the variable or nested property doesn't exist, the value defaults to `null`. The same applies if you do not provide a `source`.
```
